From: David Faure <faure@kde.org>
Date: Wed, 01 Jan 2014 12:57:28 +0000
Subject: This needs kglobalaccel:
X-Git-Url: http://quickgit.kde.org/?p=kde-workspace.git&a=commitdiff&h=1baf101e2b33dc2cb9d239df19eab1a73e2f0917
---
This needs kglobalaccel:

./ksldapp.cpp:39:#include <KGlobalAccel>
./ksldapp.cpp:125:        KGlobalAccel::self()
---


--- a/ksmserver/screenlocker/CMakeLists.txt
+++ b/ksmserver/screenlocker/CMakeLists.txt
@@ -31,6 +31,7 @@
 target_link_libraries(screenlocker_static kworkspace
    Qt5::X11Extras
    KF5::IdleTime
+   KF5::GlobalAccel
    ${X11_LIBRARIES}
    ${X11_Xcursor_LIB}
    ${XCB_XCB_LIBRARY}

